builds.yml

```diff
@@ -145,6 +147,12 @@ features:
       - BLOCKAID_PUBLIC_KEY: 066ad3e8af5583385e312c156d238055215d5f25247c1e91055afa756cb98a88
   petnames:
   conf-redesign:
+  notifications:
+    env:
+      - AUTH_API: https://authentication.api.cx.metamask.io
```

Comment:
Is there a reason these are set only for the main and flask build types, not for all of them?

Reply:
These are our main targets. However not us against supporting all targets.

Reply:
I've discussed with the team. We'll initially support notifications for Main and Flask.
If the extension team prefers we support all, then that can be done (not a hard requirement)